src/SDKs/_metadata/datalake-analytics_data-plane.txt

```diff
@@ -1,10 +1,11 @@
-2018-02-07 22:22:52 UTC
+2018-05-02 05:55:28 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: Azure
 Branch:      master
-Commit:      f6330512616c264543fec0f02f21072a1559173b
+Commit:      2ae4511d8575f4f9efae0b0c144ef4f5adc8546f
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    C:\Users\jooro\AppData\Roaming\npm `-- autorest@2.0.4245  
+Bootstrapper version:    D:\code\nodejs\node_global `-- autorest@2.0.4262  
```

Comment:
Not sure why this file is being updated, @idear1203 which `generate.cmd` was run to generate the code

Reply:
I use this `generate.cmd`: [src/SDKs/DataLake.Analytics/Management.DataLake.Analytics/generate.cmd](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/DataLake.Analytics/Management.DataLake.Analytics/generate.cmd).

This file seems updating frequently.